### ISSUE TITLE
How to add Firmware upload view

### ISSUE THREAD

**Issue by mattisalo** [collin80/SavvyCAN#240]
Bit new to these sources... How to get Firmware uploader to be visible on windows version?

**Comment by collin80** [comment]
If you're using the WIP branch then these lines start at 192 in MainWindow.cpp:

    //these either are unfinished/not working or are not for general use. But,they exist
    //so if you want to enable them and play with them then go for it.
    ui->actionFirmware_Update->setVisible(false);
    ui->actionMotorControlConfig->setVisible(false);
    ui->actionSingle_Multi_State_2->setVisible(false);

Change whichever you want to true and recompile then you'll see that window as an option. I haven't tried the firmware updater in quite a while. Bit rot might have gotten to it.

**Comment by mattisalo** [comment]
THX ,i got it enabled. But when i push the "start upload" nothing happens. I'm trying to use PCan as canbus adapter. No can messages are sent. Bo something really has happened to it....

**Comment by collin80** [comment]
Yeah, sounds like it is broken. I recently re-factored the whole CAN bus system and that probably broke the firmware uploader window. I don't ever use it anymore so it wasn't a big priority for me to get it fixed. In fact, I don't know if I even have any hardware laying around to test with at the moment. But, I'll put it on the list of things to fix.
